### ISSUE TITLE
How to get export selector

### ISSUE THREAD

**Issue by ro3rr** [FUE5BASE/FUE5-Exporter#1]
There was no hint about how to get the selector item, I had to go through the source code to find out that it was a giveable item.(/c game.player.insert"FUE5Exporter__export-selector") You should specify it in the readme

**Comment by Atria1234** [comment]
Yes, my bad. There is also a Factorio shortcut which gives it to you

**Comment by Blogshot** [comment]
Voting to reopen this.

There is no indication of any shortcut in the Readme and I couldn't find the shortcut you mentioned anywhere in the settings.

I looked recursively in each category under Mod-Settings as well as the Control Settings. I had to use the console command from ro3rr to get the item.

**Comment by Atria1234** [comment]
Yes, it will be in next version, when I'm done with rewrite of load logic in UE5
